style.css

```diff
@@ -95,10 +184,89 @@ h3 {
   width: 20px;
 }
 
+.icon-lg {
+  width: 50px;
+}
+
 /* List style */
 
 .xul {
   display: flex;
   list-style-type: none;
   padding: 0;
 }
+
+.flex-column {
+  flex-direction: column;
+}
+
+/* Box rounds */
+
+.border-card {
+  border-radius: 30px;
+  border: 1px solid #dfe1e6;
+}
+
+.border-10px {
+  border-radius: 10px;
+}
+
+.border-bl {
+  border-radius: 0 0 0 100px;
+}
+
+.border-tr {
+  border-radius: 0 100px 0 0;
+}
+
+.border-bottom-line {
+  border-bottom: 1px solid #dfe1e6;
+}
+
+/* Background settings */
+
+.bg-hero {
+  background: url(assets/images/Background.svg) no-repeat center center;
+}
+
+.bg-box {
+  background-color: #fff;
+}
+
+.bg-tag {
+  background-color: #ebebff;
+}
+
+.bg-list {
+  background-color: #f7f7f9;
+}
+
+/* Grid */
+
+.grid-container-card {
+  display: grid;
+  grid-template-rows: auto auto auto auto auto auto;
+}
+
+.grid-container-list {
+  display: grid;
+  row-gap: 10px;
+}
+
+.grid-container-row {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  column-gap: 10px;
+}
+
+/* Button styling */
+
+button {
+  font-size: 17px;
+  color: #396df2;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  background-color: transparent;
+  border-radius: 10px;
+  margin: 10px;
+}
```

Comment:
- [ ] I would kindly suggest you give your `See Project` as well as the `Get my resume` button some `Active`, `Hover`, and `Pressed` styling effect as shown in the[ Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=FKBvXOXX9rZtBa1A-0)

Kindly check out this for your peruse: 👇 

![btn](https://user-images.githubusercontent.com/78634841/205191872-51eebf0a-e1ce-44ca-91f9-6fd526b0f046.png)